ReactWindows/ReactNative/Animated/AdditionAnimatedNode.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+
+namespace ReactNative.Animated
+{
+    class AdditionAnimatedNode : ValueAnimatedNode
+    {
+        private readonly NativeAnimatedNodesManager _manager;
+        private readonly int[] _inputNodes;
+
+        public AdditionAnimatedNode(int tag, JObject config, NativeAnimatedNodesManager manager)
+            : base(tag)
+        {
+            _manager = manager;
+            _inputNodes = config.GetValue("input").ToObject<int[]>();
```

Comment:
JSON validation should be done in JavaScript.

---
In reply to: [74442452](https://github.com/ReactWindows/react-native-windows/pull/583#discussion_r74442452) [](ancestors = 74442452)